zoho.go

```diff
@@ -50,14 +53,30 @@ func (s *ZohoService) AddRow(tableUri string, columnValues map[string]string) (*
 
 	var addedRows ZohoAddRowResponse
 
-	err := s.sendAPIRequest(columnValues, tableUri, addRow, &addedRows)
+	err := s.sendAPIRequest(columnValues, tableUri, addRowAction, &addedRows)
 	if err != nil {
 		return nil, errors.Wrap(err, errMessage)
 	}
 
 	return &addedRows, nil
 }
 
+// ImportCSV import a bulk of rows as CSV
+func (s *ZohoService) ImportCSV(tableUri, csvData string, config map[string]string) (*ZohoAddRowResponse, error) {
+	const errMessage = "could not import data in zoho"
+	config["ZOHO_IMPORT_DATA"] = csvData
+	config["ZOHO_IMPORT_FILETYPE"] = csvFileType
+	config["ZOHO_AUTO_IDENTIFY"] = autoIdentify
+
+	var resp ZohoAddRowResponse
```

Comment:
The declaration of the variable (as well as the declaration of the const error message above) should not be cuddling with the rest of the code.